src/utils/thread_helper.hpp

```diff
@@ -69,7 +70,12 @@ namespace agent {
 class ThreadHelper
 {
 public:
-    using DeviceRoleHandler       = std::function<void(otDeviceRole)>;
+    using DeviceRoleHandler = std::function<void(otDeviceRole)>;
+
+#if OTBR_ENABLE_DHCP6_PD
+    using Dhcp6PdStateCallback = std::function<void(otBorderRoutingDhcp6PdState)>;
+#endif // OTBR_ENABLE_DHCP6_PD
```

Comment:
nit: such a small block of `#if` is unnecessary to be commented

Reply:
For the macro-guarded ones, maybe you can put them behind the others.

Reply:
which macro-guarded one?